src/backend/config/settings.py

```diff
@@ -372,6 +372,20 @@ def get(self, path: str) -> Any:
                 return None
         return value
 
+    def safe_lookup(self, *args, start=None, default=None):
```

Comment:
Looks like we have a new `get` function that is similar; let's use that one instead.

Reply:
Just got to this point - yup let's use get!